test/404s.test.js

```diff
@@ -1772,3 +1772,26 @@ test('setNotFoundHandler should be chaining fastify instance', t => {
 
   t.end()
 })
+
+test('Should fail to invoke callNotFound inside a 404 handler', t => {
+  t.plan(2)
+
+  let fastify = null
+  try {
+    fastify = Fastify({
+      frameworkErrors: (error, req, reply) => {
+        return reply.callNotFound(error)
```

Comment:
could you add a test adding a not found handler?
https://github.com/fastify/fastify/blob/main/docs/Reference/Server.md#setnotfoundhandler

I would expect it is executed

Reply:
I thought it is handled in `Should fail to invoke callNotFound inside a 404 handler` test case. Should i write another testcase ?

Reply:
Yes please. That case does not contain a custom not found handler but it relies on the default one